blog post for 2026 goal 2 (#29)
swcarpentry/governance: blog/2026-03-goal2.md

# Software Carpentry Governance Committee - Goals for 2026
The Software Carpentry Courses are an excellent start for beginners to

> get into scientific programming and managing their own data. However,

Reviewer: ```suggestion
The many years of running  Software Carpentry workshops have shown that this workshop is an excellent start for beginners to
get into scientific programming and managing their own data. However,
```

> a common question afterwards is what *course can we take

Reviewer: ```suggestion
a common question afterwards is what *workshop can we take
```
standardizing terminology to use carpentries "workshop" instead of "course"

> next?* The Software Carpentry Governance Committee would like to

Reviewer: ```suggestion
next?* The Software Carpentry Governance Committee aims to
```

address this question by adding at least one intermediate programming

> course to the [SWC

Reviewer: ```suggestion
workshop to the [SWC
```

> curriculum](https://software-carpentry.org/lessons/). 

Reviewer: ```suggestion
curriculum](https://software-carpentry.org/lessons/) in 2026. 
```

Two courses from the [Carpentries
Incubator](https://carpentries-incubator.org/) are contenders:

* [Building Better Research
  Software](https://carpentries-incubator.github.io/better-research-software/)
* [Intermediate Research Software

>   Development](https://carpentries-incubator.github.io/python-intermediate-development/)

Reviewer: I'm undecided if we want to mention this in the blog post at present.

> Both courses address the *What next?* question and are under beta

Reviewer: ```suggestion
Both workshops address the *What next?* question and are under beta
```

> testing. Participants learn how to deal with larger and more complex

Reviewer 2: - https://carpentries.org/blog/2025/08/building-better-beta-pilots
- https://carpentries.org/blog/2022/03/incubator-lesson-spotlight-intermediate-research-software-development-in-python

programming problems by being introduced to the software life cycle
including planning, testing and releasing software and collaborative
development. 

While assessing which courses to add to the official curriculum the
SWC Governance Committee will document and publish this process. This
will allow us to push some more of the excellent courses from the
incubator to official curriculum. 

> We are also very much aware of the discussions of moving away from big

Reviewer 2: I'd suggest we can have an explicit **"Call for Participation"** from the community similar to calls for [pilot workshops](https://carpentries.org/blog/2025/10/geospatial-data-carpentry-for-urbanism) and [AI Carpentry](https://carpentries.org/blog/2025/08/genai-curriculum-discussions).

> corporate technology. There are other good reasons apart from digital

Reviewer: ```suggestion
We are also very much aware of the discussions of moving away from big
corporate technology, such as the dependency on GitHub as a host for git repositories. There are other good reasons apart from digital
```

sovereignty for organisations to use gitlab to host their own code

> repositories. Although not a priority for this year, the SWC

Reviewer 2: - https://eurostack.eu/the-letter
- https://sfconservancy.org/GiveUpGitHub
- https://openfuture.eu/blog/european-open-digital-ecosystems-and-digital-commons

Governance Committee would be very supportive if the community were to
develop a gitlab alternative to the [Version control with
git](https://swcarpentry.github.io/git-novice/) lesson and publish it

> in the incubator.

Reviewer: ```suggestion
repositories. The SWC
Governance Committee would be very supportive if community members were to
develop a gitlab alternative to the [Version control with
git](https://swcarpentry.github.io/git-novice/) lesson and publish it
in the incubator. The committee would be interested in reviewing 
that new lesson for consideration once we've established a process for new lesson/workshop adoption into SWC.
```

Reviewer: we may also want to add here a specific call to action - like if you are interested, email Sarah and she will organize a group to work on this task?

including planning, testing and releasing software and collaborative
development.

While assessing which courses to add to the official curriculum the
SWC Governance Committee will document and publish this process. This
will allow us to push some more of the excellent courses from the
incubator to official curriculum.

If you have any comments regarding the above selection or have suggestions for new courses that would be good contenders, please reach out to the committee at: swc-governors@lists.carpentries.org

## Call for Lesson Development - Version Control with Git

We are very aware of the discussions of moving away from big
corporate technology, such as the dependency on GitHub as a host for git repositories. There are [many good reasons](https://sfconservancy.org/GiveUpGitHub), including [digital
sovereignty for organisations](https://edri.org/our-work/europes-digital-sovereignty-starts-with-open-source), to use GitLab to host their own code
repositories.

The SWC Governance Committee would be happy to support community members that would like develop a GitLab alternative to the [Version Control with
Git](https://swcarpentry.github.io/git-novice/) lesson and publish it

> in the Incubator. We will review new lesson for consideration after finalising the process for new lesson/workshop adoption into SWC.

Reviewer: ```suggestion
in the Incubator. We will review new lessons for consideration after finalising the process for new lesson/workshop adoption into SWC.
```
